### ISSUE TITLE
Tooltip / Popover: WCAG 2.1 - Content should stay visible on hover

### ISSUE THREAD

**Issue by sprungmarker** [twbs/bootstrap#28411]
For WCAG 2.1 - 1.4.13 Content on Hover or Focus
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

Content area for tooltips und popover should stay visible on hover i.e. with a mouse.
Especially for low vision users this is important.

Example WAI: https://codepen.io/Moiety/pen/LaPvWy

**Comment by patrickhlauke** [comment]
and it should be dismissible without needing to move the mouse/pointer

**Comment by sprungmarker** [comment]
@patrickhlauke yes i.e. ESC

**Comment by viktomas** [comment]
This would be also very useful when having buttons or links in the popover content.

**Comment by RyanBerliner** [comment]
Is this something the team would be open to including in a v5 release? Making the tooltips both _hoverable_ and _dismissable_ to satisfy 1.4.13 is something I'd love to help get done. Both can be accomplished in a non-breaking fashion. Let me know and I can get a pr together if you all are on board.

**Comment by patrickhlauke** [comment]
@RyanBerliner if you want to have a crack at this, sure...go for it
